cloudinit/cmd/devel/hotplug_hook.py

```diff
@@ -31,15 +31,31 @@ def get_parser(parser=None):
         parser = argparse.ArgumentParser(prog=NAME, description=__doc__)
 
     parser.description = __doc__
-    parser.add_argument("-d", "--devpath", required=True,
-                        metavar="PATH",
-                        help="sysfs path to hotplugged device")
-    parser.add_argument("-s", "--subsystem", required=True,
-                        help="subsystem to act on",
-                        choices=['net'])
-    parser.add_argument("-u", "--udevaction", required=True,
-                        help="action to take",
-                        choices=['add', 'remove'])
+    parser.add_argument(
+        "-s", "--subsystem", required=True,
+        help="subsystem to act on",
+        choices=['net']
+    )
+
+    subparsers = parser.add_subparsers(dest='hotplug_action')
+
```

Comment:
I think you need a `    subparsers.required = True` to make sure we choose either query or handle. otherwise we get this trace:
```bash
root@dev-b:~# cloud-init devel hotplug-hook -s net
AttributeError: 'Namespace' object has no attribute 'devpath'
```

```suggestion
    subparsers.required = True

```